src/components/careers/Positions/index.tsx

```diff
@@ -54,12 +54,12 @@ const Section = styled.section`
   }
 `
 
-const Positions = ({ positions }: { positions: Job[] | undefined }) => {
+const Positions = ({ positions }: { positions: Job[] }) => {
```

Comment:
Are we sure it's never undefined? What if the API returns some other object?

Reply:
The [state is initially an empty array](https://github.com/safe-global/safe-landing-page/blob/dev/src/components/careers/Positions/usePositions.ts#L7) and it returns an empty array if there are no positions (see example [here](https://legible.breezy.hr/json)).